components/molecules/UniversalBenefitCard.js

```diff
@@ -0,0 +1,37 @@
+import propTypes from 'prop-types'
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+import { ProgramCodes, StatusCodes } from '../../objects/UniversalBenefit'
+import BenefitCardHeaderSummary from './BenefitCardHeaderSummary'
+
+export default function UniversalBenefitCard(props) {
+  const t = props.locale === 'en' ? en : fr
+
+  return (
+    <div className="benefit-card">
+      {/* Status Bar Component goes here */}
+      <h2>{t[props.benefit.programCode]}</h2>
+      {props.benefit.summaries == null || props.benefit.summaries.length <= 0
```

Comment:
When checking for null, I think you can do something like `props.benefit.summaries` and if it's not null, it will default to true.

i think you can remove the `== null`.

Reply:
But I'm looking for a true result when it is null; removing it gives me the opposite; so it is needed here.

Reply:
and we actually want this to be a double equals so that if summaries is null or undefined we get a true.

Reply:
Sounds good with me.